packages/daimo-common/src/daimoLink.ts

```diff
@@ -131,9 +139,20 @@ function formatDaimoLinkInner(link: DaimoLink, linkBase: string): string {
     case "tag": {
       return `${linkBase}/t/${link.id}`;
     }
+    case "error": {
+      return `${linkBase}/error`;
+    }
```

Comment:
i don't think this needs to be its own link type?

what would be the point of deeplinking to https://daimo.com/l/error ?